searx/engines/bing_news.py

```diff
@@ -31,69 +42,71 @@
 base_url = 'https://www.bing.com/'
 search_string = 'news/search?{query}&first={offset}&format=RSS'
 search_string_with_time = 'news/search?{query}&first={offset}&qft=interval%3d"{interval}"&format=RSS'
-time_range_dict = {'day': '7',
-                   'week': '8',
-                   'month': '9'}
-
+time_range_dict = {
+    'day': '7',
+    'week': '8',
+    'month': '9'
+}
 
-# remove click
 def url_cleanup(url_string):
+    """remove click"""
+
     parsed_url = urlparse(url_string)
     if parsed_url.netloc == 'www.bing.com' and parsed_url.path == '/news/apiclick.aspx':
         query = dict(parse_qsl(parsed_url.query))
-        return query.get('url', None)
+        url_string = query.get('url', None)
     return url_string
 
-
-# replace the http://*bing4.com/th?id=... by https://www.bing.com/th?id=...
 def image_url_cleanup(url_string):
+    """replace the http://*bing.com/th?id=... by https://www.bing.com/th?id=..."""
+
     parsed_url = urlparse(url_string)
-    if parsed_url.netloc.endswith('bing4.com') and parsed_url.path == '/th':
+    if parsed_url.netloc.endswith('bing.com') and parsed_url.path == '/th':
```

Comment:
AFIU, this is where there is a functional change. Am I right?

Reply:
> AFIU, this is where there is a functional change. Am I right?

right! .. sorry, I should have this PR in two commits.